src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  font-family: Avenir, serif;
+}
```

Comment:
Not fixed. It works for html tag
![2023-01-18_14h36_21](https://user-images.githubusercontent.com/61904995/213173241-7ebcd300-7866-4a85-9c13-b343e5cd0576.png)